lib/cli/src/frameworks/lit/ts/Button.stories.ts

```diff
@@ -0,0 +1,51 @@
+import './Button';
+import { html } from 'lit';
```

Comment:
```suggestion
import { html } from 'lit';

import './Button';
```

Reply:
Can you make sure to add extensions to the button import.  
`import './Button.js'`